cengage/react-magma
# Hyperlink: Support Icons and toggling underline styling

Issue: #1120

## What I did

- Added support for an icon on the left, right, or both sides of a Hyperlink
- Added `hasUnderline` prop that defaults to true
- Updated Hyperlink and native anchors hover color

## Screenshots:

![image](https://github.com/cengage/react-magma/assets/91160746/44be3b7e-e779-4807-84e7-ac78caffd5fe)
![image](https://github.com/cengage/react-magma/assets/91160746/803147f0-796a-430d-98c8-718a2fdc698f)


## Checklist 

## How to test

Confirm all use cases for hyperlinks are working. Consider:
- styled as button or link
- inverse on/off
- hasUnderline on/off
- passing 1 icon
- passing 2 icons
- icon positions: left, right or both
- using it inside `<Paragraph>`, `<p>`, and other text wrappers
